client/app/_services/tests-sessions/tests-sessions.service.js

```diff
@@ -22,11 +22,16 @@ const testsSessionsService = function testsSessionsService(
         resetCachedParams,
         fetchAdditionalSearchParams,
         getSessionById,
+        refactorPlatformData,
 
         get activeParams() { return lastParams; },
         set activeParams(newSC) { lastParams = newSC; return true; },
     };
 
+    function refactorPlatformData(data) {
+        return [data.browserName.toLowerCase(), data.version, data.osName.toLowerCase()];
```

Comment:
Please, do not convert to lower case `osName`

Reply:
fixed